docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
I originally considered that naming convention, but, I was paranoid.  What if someone named a store "default"?  I did not want to suggest a change that could adversely impact an existing deployment.  That is why I placed the "default" at the end.  A compromise might be something like:
    stores.__default__.changelog.replication.factor
Still possible for a name collision, but, a lot more unlikely (and potentially deserved).

Paranoia aside, I am wide open to suggestions on the naming convention to use.  I just want to get a name established so that we can patch that name back to one of our private branches based on 0.11 or 0.12.